_posts/2025-05-20-firefox-security-privacy-newsletter-2025-q1.md

```diff
@@ -0,0 +1,72 @@
+
+**Safebrowsing updates**: Starting with Firefox 138, Safebrowsing now classifies only top-level loads ([Bug 1953088](https://bugzilla.mozilla.org/show_bug.cgi?id=1953088)) and no longer blocks subresource loads, like e.g. external JS scripts, or also images, which reduces the risk of web-compat issues caused by unsupported subresource blocking.
+
+**Unbreaking websites from Anti-Tracking mechanisms**: We have worked with [disconnect.me](http://disconnect.me) to unblock certain consent management providers (CMP) in private browsing mode and ETP-Strict (see non-exhaustive [bug list](https://bugzilla.mozilla.org/buglist.cgi?bug_id=1909809%2C1906427%2C1909418%2C1942290%2C1951065%2C1924998%2C1936252%2C1934494&list_id=17532862)). CMPs \- often seen as “Cookie Banners” \- are responsible for almost half of all site breakage when they are blocked by our anti tracking protection. This work allowed many websites to work as expected again in private browsing mode.
+
+**Smartblock Embeds gives users control over blocked content**: Starting with Firefox 136, SmartBlock Embeds support empowers ETP-Strict and PBM users to consciously opt into highly visible content, and prevents the perception that the page is ‘broken’ \- even though our protections are working as intended. Smartblock works for embedded content from Instagram ([Bug 1892173](https://bugzilla.mozilla.org/show_bug.cgi?id=1892173)), TikTok ([Bug 1892172](https://bugzilla.mozilla.org/show_bug.cgi?id=1892172)) and Twitter/X ([Bug 1901602](https://bugzilla.mozilla.org/show_bug.cgi?id=1901602)), which are blocked by default in Private windows or ETP-Strict mode. Embedded blocked content from these domains will show a 1-button click option for users to un-block all content from that domain on that specific website.
+
+**Away from DNT and towards GPC**: In Q1, Firefox became the first major browser to remove the “Do Not Track” checkbox from the privacy preferences/settings, recognising that Global Privacy Control (GPC) is a better option for 2025\. See [SUMO article](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature) for details.
+
+## Core Security
+
+**Hardening Firefox:** Starting in Firefox 135, Firefox UI uses a strict Content-Security-Policy (CSP) in [browser.xhtml](https://searchfox.org/mozilla-central/source/browser/base/content/browser.xhtml) to prevent XSS-like sandbox escapes ([Bug 1935985](https://bugzilla.mozilla.org/show_bug.cgi?id=1935985)). This involved modifying over 600 inline event handlers and a careful rollout with notice to people who run modified Firefox configurations. A detailed description of these security hardening efforts are available in our blogpost: [Hardening the Firefox Frontend with Content Security Policies](https://attackanddefense.dev/2025/04/09/hardening-the-firefox-frontend-with-content-security-policies.html).
```

Comment:
I don't think people know what browser.xhtml is and would just over-generalize a bit and say "for the Firefox UI"